docker/web.prod.env.example

```diff
@@ -0,0 +1,6 @@
+SECRET_KEY=not-so-secret
+ALLOWED_HOSTS=*
+FRONTEND_SENTRY_DSN=https://sentry-random-hash@sentry.io/12345
```

Comment:
The sentry integration should also be upgraded, this is the legacy way. In the new client I think it's not so bad to store the real DSN on GitHub

Reply:
Right, forgot about Sentry; I'll upgrade it in a separate PR.